app/src/main/java/org/dhis2/usescases/login/accounts/ui/AccountsScreen.kt

```diff
@@ -96,16 +96,13 @@ fun LoginHeader() {
     Column(
         Modifier
             .fillMaxWidth()
-            .padding(16.dp)
+            .padding(16.dp, 8.dp, 16.dp, 12.dp)
     ) {
         Row(
             horizontalArrangement = Arrangement.Center,
             modifier = Modifier.fillMaxWidth()
         ) {
-            Image(
-                painter = painterResource(id = R.drawable.ic_dhis_white),
-                contentDescription = null
-            )
+            AndroidView(factory = { View.inflate(it, R.layout.dhis_logo, null) })
```

Comment:
Did you check if this worked?
```
Image(
            imageVector = ImageVector.vectorResource(id = R.drawable.ic_dhis_white),
            contentDescription = ""
        )
```

Reply:
Yes, I tried with different painters and resources and with the ImageVector and it was displayed the same. Using the androidView was the only way to show it good. We need to take a deeper look at this, but for now this solution work.